dedicated_rack_test.go

```diff
@@ -2834,7 +2835,7 @@ func TestDedicatedRackListDataTrafficNotificationSettings(t *testing.T) {
 	assert.Equal(resp.Settings[0].Frequency, "WEEKLY")
 	assert.Equal(resp.Settings[0].Id, "12345")
 	assert.Empty(resp.Settings[0].LastCheckedAt)
-	assert.Equal(resp.Settings[0].Threshold, "1")
+	assert.Equal(resp.Settings[0].Threshold, json.Number("1"))
```

Comment:
@msslr let's change these lines to 
`assert.Equal(resp.Settings[0].Threshold.String(), "1")`